Review 'Upgrading FastStore to v4' guide (#2646)
vtexdocs/dev-portal-content: docs/faststore/docs/getting-started/upgrading-faststore-to-v4.mdx

Reviewer: We need to update this part with this instructions: https://vtex.slack.com/archives/C02JMH9AYR1/p1778251299087769?thread_ts=1778189498.823789&cid=C02JMH9AYR1

#### Generate and upload the schema

After ensuring your CMS files are in the new format, generate and upload the schema:

1. Log in to your VTEX account:

  ```shell
  vtex login {accountName}
  ```

2. Generate the schema:

  ```shell
  vtex content generate-schema cms/faststore/components cms/faststore/pages -o cms/faststore/schema.json
  ```

3. Upload the schema:

  ```shell
  vtex content upload-schema cms/faststore/schema.json
  ```

The table below explains the command arguments:

| Argument | Description |
| -------- | ----------- |
| `cms/faststore/components` | Directory containing your component schema files. |
| `cms/faststore/pages` | Directory containing your Content Type schema files. |
| `-o cms/faststore/schema.json` | Local output path for the generated schema file before upload. |

For more details on these commands, see [Content plugin](https://developers.vtex.com/docs/guides/content-plugin).

#### Stores using CMS without custom components

> If your store does not have any custom CMS schema files yet and should use only the default FastStore schema, you do not need to run `generate-schema`. Instead, create a `schema.json` file and add the following content:

Reviewer: acho que é bom destacar essa parte 👍

> If your store does not have any custom CMS schema files yet and should use only the default FastStore schema, you do not need to run `generate-schema`. Instead, create a `schema.json` file and add the following content:

Author: ```suggestion
If your store **does not have any custom CMS schema files** yet and should use only the default FastStore schema, you do not need to run `generate-schema`. Instead, create a `schema.json` file and add the following content:
```

> If your store **does not have any custom CMS schema files** yet and should use only the default FastStore schema, you do not need to run `generate-schema`. Instead, create a `schema.json` file and add the following content:

Reviewer: ```suggestion
If your store **does not have any custom CMS schema files** yet and should use only the default FastStore schema, you do not need to run `generate-schema`. Instead, create a `schema.json` file in `cms/faststore` and add the following content:
```

  ```json
  {

>     "$base": "vtex.faststore@4.0.0"

Reviewer: ```suggestion
    "$base": "vtex.faststore"
```

vamo deixar só assim p/ pegar a última versão atualizada mesmo :)

Reviewer: ai lá no note trocamos, acho que é melhor.

  }
  ```

Upload this content by running the following command in the terminal:

  ```shell
  vtex content upload-schema cms/faststore/schema.json
  ```

This creates a schema for your store in CMS that points to the FastStore base schema.

> > ℹ️ You can also use `"vtex.faststore"` as the `$base` value if you do not want to pin the version explicitly.

Reviewer: ```suggestion
> ℹ️ You can also use `"vtex.faststore@4.0.0"` as the `$base` value if you want to pin the version explicitly, otherwise "vtex.faststore" for the latest version.
```

> > ℹ️ You can also use `"vtex.faststore@4.0.0"` as the `$base` value if you want to pin the version explicitly, otherwise "vtex.faststore" for the latest version.

Author: ```suggestion
> ℹ️ You can also use `"vtex.faststore@4.0.0"` as the `$base` value if you want to pin the version explicitly. Otherwise, use `"vtex.faststore"` to get the latest version.
```
